AI_USAGE_LOG.md

```
@@ -0,0 +1,161 @@
+   present and runnable.
+3. **`browserType.launch: Executable doesn't exist`** — Playwright's browser
+   binaries weren't downloaded yet. Fixed with `yarn playwright install
+chromium`.
+
+## 5. A real bug found by the new test, not assumed
+
+The first real test run failed on the new
+`assigning a budgeted amount updates the category and the total budgeted`
+test:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)